src/glo_outline.ts

```diff
@@ -56,6 +58,11 @@ export function glossaryGetSymbols(text: string): vscode.DocumentSymbol[] {
                 );
             }
         }
+        // Don't add the letter if it is a "no letter" and it doesn't actually have children.
```

Comment:
So we should only do this if there are no entries? (for that part of the file)

Reply:
You mean the condition should be `children.length != 0` instead of `letters[idx].letter != LetterNoLetter.letter || children.length != 0`?

Reply:
No, the condition looks correct, I was just checking that I've understood when this "branch" is run! Is it max once per file?

Reply:
Yes!